### PR TITLE
add input

### PR DESCRIPTION
**Pull request by dev-nes-ilya**
added input that can change the title

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,11 +3,24 @@ import logo from './logo.svg';
 import './App.css';
 
 class App extends Component {
+  state = {
+    tasks: 'первая задача'
+  };
+  
+
+  onChangeInput = (evt) => {
```
Comment (denistakeda) [inline src/App.js]: Looks good, but something wrong with formatting. Please try to use some autoformat tools, like https://github.com/prettier/prettier

File: src/App.js [hunk 2]
```diff
@@ -3,11 +3,24 @@ import logo from './logo.svg';
...
+   })
+  };
+
+
   render() {
     return (
       <div className="App">
         <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
+          {/* <img src={logo} className="App-logo" alt="logo" />
```
Comment (denistakeda) [inline src/App.js]: Please don't leave commented code. You can always restore any code from git history, so if you don't need this one, just remove it

File: src/App.js [hunk 3]
```diff
@@ -17,9 +30,14 @@ class App extends Component {
...
             rel="noopener noreferrer"
           >
-            Learn React 123
-          </a>
+          </a> */}
+          <tasks>
+            cheklist: {this.state.tasks}
+          </tasks>
+          <input type="text" onChange={this.onChangeInput}></input>
+          {/* <button onClick={this.onClickButton}>add</button> */}
```
Comment (denistakeda) [inline src/App.js]: Same here

File: src/App.js [hunk 4]
```diff
@@ -17,9 +30,14 @@ class App extends Component {
             target="_blank"
             rel="noopener noreferrer"
           >
-            Learn React 123
-          </a>
+          </a> */}
+          <tasks>
```
Comment (denistakeda) [inline src/App.js]: There is no such tag as `tasks`. Please use `div` instead.

**Review by denistakeda — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (denistakeda) [reply src/App.js]: Got Worse

File: src/App.js [hunk 5]
```diff
@@ -1,28 +1,28 @@
...
+					<div>cheklist: {this.state.tasks}</div>
+					<input type="text" onChange={this.onChangeInput} />
+				</header>
+			</div>
+		);
+	}
 }
 
-export default App;
+export default App;yarn
```
Comment (denistakeda) [inline src/App.js]: Something that not belongs here.

**Review by denistakeda — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 5 above)
Comment (dev-nes-ilya) [reply src/App.js]: I understand

File: src/App.js [hunk 6]
```diff
@@ -1,24 +1,24 @@
...
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React 123
-          </a>
+          <div>cheklist: {this.state.tasks}</div>
+          <input type="text" onChange={this.onChangeInput} />
```
Comment (denistakeda) [inline src/App.js:21]: you need also `value` property here, otherwise, you can't clear the input later.

**Review by denistakeda — CHANGES_REQUESTED** [review]

**Pull request merged**